workspaces/ts-source-parser/package.json

```
@@ -0,0 +1,37 @@
+{
+    "name": "@nodesecure/ts-source-parser",
```

```suggestion
  "name": "@nodesecure/ts-source-parser",
```
We indents with 2 spaces (need to update the whole file). If you use VSC do you have EditorConfig extension enabled ?